### ISSUE TITLE
documenting vcr purpose

### ISSUE THREAD

**Issue by ssbarnea** [jctanner/ansible-vcr#1]
I think it would be very useful to add a sentence or paragraph long, description of the vcr purpose.

I am asking this as I found it while trying to figure out this is related or not to https://github.com/ansible/ansible/issues/38786

**Comment by jctanner** [comment]
@ssbarnea I agree. I'll work on that more once I get all the weird edge cases sorted out. Whether or not it will be useful for 38786, I don't know. My initial usecase is LARGE+COMPLICATED playbooks with HUGE inventories that have performance impacts on the controller and we need to have a way to replicate without paying out the nose for EC2 bills.

**Comment by ssbarnea** [comment]
@jctanner Thanks for the update, I will be watching the progress. If performance is what you are looking for you may want to have a look at http://mitogen.readthedocs.io/en/latest/ansible.html - it did work well for me but in my case inventory is usually <10, only the playbooks are large.
